release_notes/v4.0.0.md

```diff
@@ -0,0 +1,62 @@
+# v4.0.0
+
+This major version introduces a new optimized initialization sequence which changes when key events like creating an appUser or establishing a websocket connection happen. This new version also brings changes to how the Web Messenger is rendered on the page in order to minimize conflicts with the host page that may arise due to other Javascript libraries that are in use.
+
+## What's changed?
+
+### Usage
+#### Including the Web Messenger
+The Web Messenger now uses a script loader to reduce its initial footprint and to allow the global variable to be available as soon as possible. This change doesn't apply to people using `npm` to include the Web Messenger.
+
+#### Initializing the Web Messenger
+Previously, the app token was used to identify the application used for the Web Messenger and was optional if JWT-based authentication was used. This way of identifying the app is now deprecated and the app ID is now required at initialization time instead of the app token. You can retrieve this ID from the app settings page.
+
+```js
+// Before
+// Without JWT
```

Comment:
Suggest:

`// Without JWT` --> `Initialize with appToken`
`// With JWT` --> `Initialize with JWT`

and in the after:

`// Without JWT` --> `Initialize with appId`
`// With JWT` --> `Initialize with appId and JWT`